decidim-admin/app/views/decidim/admin/newsletters/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<div id="newsletters">
+  <% provide :title do %>
+    <h2><%= t ".title" %></h2>
+  <% end %>
+
+  <div class="actions title">
+    <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.newsletter.name", scope: "decidim.admin").downcase), ['new', 'newsletter'], class: 'new' %>
+  </div>
+
+
+  <table class="stack newsletters">
+    <thead>
+      <tr>
+        <th><%= t("models.newsletter.fields.subject", scope: "decidim.admin") %></th>
+        <th><%= t("models.newsletter.fields.created_at", scope: "decidim.admin") %></th>
+        <th><%= t("models.newsletter.fields.sent_at", scope: "decidim.admin") %></th>
+        <th><%= t("models.newsletter.fields.progress", scope: "decidim.admin") %></th>
+        <th class="actions"><%= t("actions.title", scope: "decidim.admin") %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @newsletters.each do |newsletter| %>
+        <tr data-newsletter-id="<%= newsletter.id %>">
+          <td><%= translated_attribute newsletter.subject %></td>
+          <td><%= l newsletter.created_at, format: :long if newsletter.created_at %></td>
+          <td><%= l newsletter.sent_at, format: :long if newsletter.sent_at %></td>
+          <td>
+            <% if newsletter.sent? %>
+              <%= "#{newsletter.total_deliveries} / #{newsletter.total_recipients}" %>
```

Comment:
Since we don't have decorators, I feel more comfortable having this in the view instead of the model. I could always create a helper, but I prefer not to pollute everything.